Source/WebKit/UIProcess/API/haiku/WebViewBase.cpp

```diff
@@ -93,5 +97,7 @@ void WebViewBase::Draw(BRect update)
 }
 void WebViewBase::MouseMoved(BPoint where,uint32 code,const BMessage* dragMessage)
 {
+	BMessage* mouseMessage = Looper()->DetachCurrentMessage();
```

Comment:
Don't detach the message here or you'll have a memory leak.
Use Looper()->CurrentMessage() instead.